_posts/2022-11-22-zero-twenty-two-release.md

```diff
@@ -0,0 +1,80 @@
+
+Let's dive right into the changes.
+
+## 🍱 BentoML Integration
+
+The BentoML integration has been on our radar for some time now and we finally took the time to flesh it out with the help of our contributors [Tim Cvetko](https://github.com/timothy102) and [Aaron Pham](https://github.com/aarnphm).
+
+The new [BentoML integration](https://zenml.io/integrations/bentoml) includes a BentoML model deployer component that lets you deploy models using major machine learning frameworks (e.g. PyTorch, Tensorflow, HuggingFace, [etc](https://www.bentoml.com/)) on your local machine and in the cloud.
+
+We will showcase the BentoML integration in our next community hour (23rd Nov 2022, 5:30PM CET). 
+Want to see it in action? Join us [here](https://zenml.io/meet).
+
+In the meantime, check out the [BentoML example]((https://github.com/zenml-io/zenml/tree/main/examples/bentoml_deployment)) on our repo.
+
+## 🚀 Airflow in the Clouds
+
+The previous Airflow orchestrator was limited to local runs and we had many additional unpleasant constraints.
+It's a pain to work with. So, we've completely rewritten the Airflow orchestrator. Now, it works both locally and with remote Airflow deployments!
+
+Watch the demo video below to see the revamped Airflow orchestrator in action.
+<iframe width="560" height="316" src="https://www.youtube-nocookie.com/embed/v-tEm4O61Y8" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+And also, check out the example of the brand-new Airflow orchestrator [here](https://github.com/zenml-io/zenml/tree/main/examples/airflow_orchestration).
+
+
+## 💔 Breaking Changes
+
+The revamped Airflow orchestrator comes with a breaking change. 
+The Airflow orchestrator now requires a newer version of Airflow and Docker installed to work.
+
+You can simply run `zenml integration install airflow` to update your installations to the correct versions.
+
+## 🪲 Notable Bug Fixes and Improvements
+
+You can now use the ZenML Label Studio integration with non-local (i.e. deployed) instances. For more information, see the [Label Studio documentation](https://docs.zenml.io/component-gallery/annotators/label-studio).
+The Label Studio [example](https://github.com/zenml-io/zenml/tree/main/examples/label_studio_annotation) walks through how you can set it up on cloud infrastructures like Azure, GCP, and AWS.
+
+We fixed the Spark [example](https://github.com/zenml-io/zenml/tree/main/examples/spark_distributed_programming) and it now works again end-to-end.
+
+We also included a fix that speeds up the data sync from the MLMD database to the ZenML server. 
+
+As usual, we also made various minor improvements which you can view [here](https://github.com/zenml-io/zenml/releases/edit/0.22.0).
+
+## 🤗 New Contributors
+
+We are grateful to have the following new contributors in this release!
+
```

Comment:
Lets acknowledge more people here and above in the bentoml section (the bentoml team and the guy who contributer to bentoml)